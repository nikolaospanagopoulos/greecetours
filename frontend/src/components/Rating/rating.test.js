
import { shallow } from "enzyme";
import React from 'react'
import Rating from "./Rating";


it('expect to render Rating Component',() => {
    expect(shallow(<Rating/>).length).toEqual(1)
})

it('expect to render Rating Component',() => {
    expect(shallow(<Rating/>)).toMatchSnapshot()
})

