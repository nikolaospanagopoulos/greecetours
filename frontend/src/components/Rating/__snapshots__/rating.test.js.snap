// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Rating Component 1`] = `
<div
  className="rating"
>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "blue",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "blue",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "blue",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "blue",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "blue",
        }
      }
    />
  </span>
  <span>
     
     
  </span>
</div>
`;
